app/src/main/java/com/thoughtworks/aceleradora/solicitacao/controller/SolicitacaoController.java

```diff
@@ -56,6 +56,7 @@ public String listaSolicitacoesDoHospital(Model model) {
     public String salvaSolicitacao(Model model, @ModelAttribute("solicitacao") Solicitacao solicitacao) {
         solicitacao.getAcompanhantes().forEach(acompanhante -> acompanhante.setSolicitacao(solicitacao));
 
+        solicitacao.setStatus("pendente");
```

Comment:
A gente podia puxar isso pro banco né? Sempre que criamos uma nova solicitação, o default é `pendente`.

O que vocês acham?

https://www.postgresql.org/docs/9.3/static/ddl-default.html
https://stackoverflow.com/questions/47974670/jpa-annotation-for-default-value-for-a-column-having-type-bigdecimal